server/src/test/java/org/elasticsearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -1069,7 +1069,7 @@ public static NodeStats createNodeStats() {
             );
         }
         RepositoriesStats repositoriesStats = new RepositoriesStats(
-            Map.of("test-repository", new RepositoriesStats.ThrottlingStats(100, 200))
+            Map.of("test-repository", new RepositoriesStats.SnapshotStats(100, 200))
```

Comment:
Could we use the full `SnapshotStats` here rather than the legacy throttling-only one? And assert its contents?